_articles/metrics.md

```diff
@@ -79,6 +80,8 @@ People are finding your project and they're using it. The next question you'll w
 
 It's never too early to start thinking about contributors. Without other people pitching in, you risk putting yourself into an unhealthy situation where your project is _popular_ (many people use it) but not _supported_ (not enough maintainer time to meet demand).
 
+The ratio of experienced, long-term contributors to recent ones also tells you about the quality of the code and need to support contributors. [Measuring your open source community's demography](https://www.oreilly.com/ideas/measure-your-open-source-communitys-age-to-keep-it-healthy) gives an overview of the project retention/engagement capability over time.
```

Comment:
> also tells you about the quality of the code and need to support contributors

Could you get more specific here? My guess is it's more about the need to support new contributors, vs. quality of code.